### PR TITLE
Weekly report

### PR DESCRIPTION
**Pull request by andrew-vancil**

### CODE REVIEW

File: weekly_data_report.R [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    d$SCREENING_DECISION[i] <- 'SCREENED IN'
+  }
+}
+
+d_missing_alleg_add <- d %>%
+  filter(address_type == 'ALLEGATION_ADDRESS',
+         is.na(address)) %>%
+  select(PERSON_ID:address_type)
+
+d_fill_in_address <- d %>%
```
Comment (cole-brokamp) [inline weekly_data_report.R:82]: Can you summarize this logic in the README.md, just so we have a clear record of it.  e.g., 

- set `SCREENED IN AR` to `SCREENED IN`
- use `CHILD_ADDRESS` if `ALLEGATION_ADDRESS` is missing

File: weekly_data_report.R [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+neighborhood_shp <- readRDS('/app/ham_neighborhoods_dep_index_shp.rds')
+
+# Overall Summary
+date_min <- min(d$DECISION_DATE)
+date_max <- max(d$DECISION_DATE)
+
+# consider 'SCREENED IN AR' same as 'SCREENED IN'
+for (i in 1:nrow(d)) {
+  if (d$SCREENING_DECISION[i] == 'SCREENED IN AR') {
+    d$SCREENING_DECISION[i] <- 'SCREENED IN'
```
Comment (cole-brokamp) [inline weekly_data_report.R:73]: Can we use something more straightforward here?  Like below where we census counts of 5 or less, we could do an `ifelse`.  

Or we could make a new variable that is logical to use for our purposes of dichotomizing the screening decision:

```
d %>%
  mutate(screened_in = SCREENING_DECISION %in% c("SCREENED IN", "SCREENED IN AR"))
```

File: weekly_data_report.R [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  filter(PERSON_ID %in% d_missing_alleg_add$PERSON_ID,
+         address_type == 'CHILD_ADDRESS') %>%
+  group_by(PERSON_ID) %>%
+  arrange(desc(ADDRESS_START)) %>%
+  slice(1) %>%
+  select(INTAKE_ID, address:dep_index)
+
+d_missing_alleg_add <- left_join(d_missing_alleg_add, d_fill_in_address, by = 'PERSON_ID')
+
+d <- d %>%
```
Comment (cole-brokamp) [inline weekly_data_report.R:92]: Let's make the addresses that are missing their own category for the summary output.  We can use a fake neighborhood name of "Missing".  This will let us know how many were not geocoded to make sure something isn't happening with the data as we automate this in the future.

**Review by cole-brokamp — COMMENTED** [review]
Looking good, I commented for some specific questions.

Additionally, can you please make sure that all of these "edge cases" we are coding in (like re-leveling screening decision, using child address if allegation address is missing, etc) are contained in the test file?  I suggest using the existing test file to make sure we can create the output dataset successfully, but also to make a much smaller test file with only edge cases that we can visually inspect to make sure they are working as expected.

File: weekly_data_report.R (hunk 3 above)
Comment (andrew-vancil) [reply weekly_data_report.R:92]: By missing address, do you mean missing from the start from JFS or do you mean not geocoded?

Comment (cole-brokamp) [reply weekly_data_report.R:92]: let's consider both cases to be missing

**Comment by andrew-vancil** [comment]
I think I've covered these changes.
- There's a test file with a couple of those odd cases that I worked through and it seems to be working properly
    - The only special cases I could think of were correctly categorizing Screen In AR as Screened In and using the child address if there's no allegation address. Did I miss any?
- I updated/expanded the readme
- I tidied up the screen_in variable code by adding the logical column like you described
- There's now a new neighborhood called 'Missing' which collects the missing addresses

**Review by cole-brokamp — APPROVED** [review]
